test/Driver/batch_mode.swift

```diff
@@ -0,0 +1,57 @@
+// RUN: %empty-directory(%t)
+// RUN: touch %t/file-01.swift %t/file-02.swift %t/file-03.swift %t/file-04.swift %t/file-05.swift
+// RUN: touch %t/file-06.swift %t/file-07.swift %t/file-08.swift %t/file-09.swift %t/file-10.swift
+// RUN: touch %t/file-11.swift %t/file-12.swift %t/file-13.swift %t/file-14.swift %t/file-15.swift
+// RUN: touch %t/file-16.swift %t/file-17.swift %t/file-18.swift %t/file-19.swift %t/file-20.swift
+// RUN: touch %t/file-21.swift %t/file-22.swift %t/file-23.swift %t/file-24.swift %t/file-25.swift
+// RUN: touch %t/file-26.swift %t/file-27.swift %t/file-28.swift %t/file-29.swift %t/file-30.swift
+// RUN: %swiftc_driver -enable-batch-mode -driver-show-job-lifecycle -driver-skip-execution -j 10 %t/file-*.swift | %FileCheck %s -check-prefix=SEED0
```

Comment:
Nitpick: I wouldn't want to rely on `*` producing files in either alphabetical or creation order. You should probably just list them out.

Reply:
Eek! google "Python glob" => "results are returned in arbitrary order", indeed.

Reply:
I don't think we guarantee Python execution either. lit.py has a mode where it just immediately shoves everything into a shell, and I *think* that's the default on both macOS and Linux.